network/model.go

```diff
@@ -28,12 +29,31 @@ type CreateGameResponse struct {
 	Actions         []model.PlayerAction                 `json:"actions"`
 }
 
+func NewGameResponse(g model.Game) GameResponse {
```

Comment:
🔮 Perhaps `NewGameResponseForPlayer(g model.Game, pID model.PlayerID)`, which does scrubbing of fields that the player shouldn't/can't know

Reply:
Sure, then we'll need to add a player to the game request struct. Theoretically, the react client will filter out the info but I guess if someone tried hard enough with curl or something they could cheat without this functionality

Reply:
Errr, on second thought, that's an http GET so there isn't a body. Hmmm I need to think about this more 🤔

Reply:
Wait, no, duh, we can just use a param in the URL to indicate which user we're requesting for. 🤦‍♂️